modules/serverless-admin-init-containers.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * /serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-admin-init-containers_{context}"]
+= Enabling init containers
+
+link:https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[Init containers] are specialized containers that are run before application containers in a pod. They are generally used to implement initialization logic for an application, which may include running setup scripts or downloading required configurations.
+
+[NOTE]
+====
+Init containers may cause longer application start-up times and should be used with caution for serverless applications, which are expected to scale up and down frequently.
+====
+
+You can enable the use of init containers for Knative services by adding the `kubernetes.podspec-init-containers` flag to the `KnativeServing` custom resource (CR).
+
+.Prerequisites
+
+* You have installed {ServerlessOperatorName} and Knative Serving on your cluster.
+
+ifdef::openshift-enterprise[]
+* You have cluster administrator permissions.
+endif::[]
+
+ifdef::openshift-dedicated[]
+* You have cluster or dedicated administrator permissions.
+endif::[]
+
+.Procedure
+
+* Add the `kubernetes.podspec-init-containers` flag to the `KnativeServing` CR:
++
+.Example KnativeServing CR
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+spec:
+  config:
+    features:
+      "kubernetes.podspec-init-containers": enabled
```

Comment:
I think the quotes here are not required. It could be just `kubernetes.podspec-init-containers: enabled`
We're not consisten in how we use quotes in this whole documentation.
I can see these example in the same doc:
* "kubernetes.podspec-volumes-emptydir": enabled
* default-external-scheme: "http"
* service-type: ClusterIP
In some test scripts we also use this:
* loglevel.kafkachannel-dispatcher: "debug"

I think all these example could leave out the quotes and use e.g. this: `kubernetes.podspec-init-containers: enabled`

Reply:
I think the rule boils down to conventions for YAML files:
Use quotes in YAML if your value includes special characters. For example, these special characters may require quotes: {, }, [, ], ,, &, :, *, #, ?, |. -, <. >, =, !, %, @, \. It is not necessary to use quotes when a single special character is surrounded by spaces, for example, * with spaces on both sides.